tests/test_fluctuation_dissipation.py

```diff
@@ -62,7 +62,8 @@ def fluctuation_dissipation_KS(M, fluctuation_method):
     ],
 )
 @pytest.mark.parametrize("hydrodynamicRadius", [0.95, 1.12])
-@pytest.mark.parametrize("numberParticles", [1,2,10])
+# @pytest.mark.parametrize("numberParticles", [1,2,10])
+@pytest.mark.parametrize("numberParticles", [10])
```

Comment:
Why did you removed the 1,2 again? Is it failing or just takes a lot of time?

Reply:
Whoops, forgot to revert that despite looking through the files! That test is quite slow for DPStokes so I often run only the 10 particle case if I'm running through tests more than once.

Reply:
One can mark tests or individual parameters with custom markers:
```python
@pytest.mark.parametrize(
    "numberParticles", [1, 2, pytest.param(10, marks=pytest.mark.slow)]
)
```
Then you can run with `pytest -m slow tests*py` or `pytest -n "not slow" tests*py`.
Technically you have to register these in a separate file, but using them as is just works for me https://docs.pytest.org/en/stable/how-to/mark.html

Something to keep in mind for the future, not worth it at this point IMO.